src/input/keys_constants.h

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifndef KEYS_CONSTANTS_H
+#define KEYS_CONSTANTS_H
+
+enum KeysMapping {
```

Comment:
```suggestion
enum class KeysMapping {
```
Can this be enum class?

Reply:
I had it enum class initially but then it made serialization of keyboard bindings into settings difficult (since enum class can't be treated as an int, which I suppose is the point)